app/views/layouts/embedded.html.haml

```diff
@@ -2,7 +2,7 @@
 %html{ lang: I18n.locale }
   %head
     %meta{ charset: 'utf-8' }/
-    = stylesheet_link_tag 'application', media: 'all'
+    = stylesheet_pack_tag 'application', media: 'all'
```

Comment:
5.1 kicks up a warning about things which aren't owned by the asset pipeline ... even if they are pointing at things which exist via webpack.